atsign-foundation/at_app_cli
# Replace dotenv with flutter_config

closes #21
closes #22 
closes #23 
closes #28 

**- What I did**
- Updated to use [at_app_flutter of the same version](https://github.com/atsign-foundation/at_widgets/pull/251)
- #21 Solved in gitignore_manager.dart
  - adds .env line to the gitignore
- #22 Solved in android_manager.dart
  - it now also sets the compileSdkVersion
- #23 Solved in create.dart
  - If it fails to add at_app_flutter the first time, there is a second retry
  - If the retry fails then the tool exits
  - This is to fix minor networking issues
- #28 Solved in android_manager.dart and pubspec_manager.dart
  - Automatically does the necessary configuration for the flutter_config package.
 Other fixes:
- Separated global variables
- Organized and relative imports for all files in the project (hence so many file changes)

**- How I did it**

Local changes

**- How to verify it**

Pull at_app 3.0.0-rc+2:

```flutter pub global activate at_app 3.0.0-rc+2```

Then generate and verify the app it produces.

**- Description for the changelog**
Replace dotenv with flutter_config.
